File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/dart/piyunior.dart
void main(List<String> args) {
  for (var i = 1; i <= 100; i++) {
    bool three = (i % 3) == 0;
    bool five = (i % 5) == 0;
    if (three && five) {
      print('fizzbuzz');
    } else if (three) {
      print('fizz');
    } else if (five) {
      print('buzz');
    } else {
      print(i);
    }
  }
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/javascript/piyunior.js
const fizzArray = async () => {
    const size = 100;
    for (let index = 1; index <= size; index++) {
        let three = (index % 3) == 0;
        let buzz = (index % 5) == 0;
        if (three && buzz) {
            console.log('fizzbuzz');
        } else {
            if (three == true) {
                console.log('fizz')
            } else if (buzz == true) {
                console.log('buzz')
            } else {
                console.log(index);
            }
        }
    }
}

fizzArray();